GDevelop.js/__tests__/GDJSAsyncCodeGenerationIntegrationTests.js

```diff
@@ -3568,4 +3571,77 @@ describe('libGD.js - GDJS Async Code Generation integration tests', function ()
       );
     });
   });
+
+  describe('Events based asynchronous functions', () => {
+    it('Resolves after awaiting an async action', () => {
+      const { gdjs, runtimeScene } = makeMinimalGDJSMock();
+      const extensionModule =
+        generateCompiledEventsForSerializedEventsBasedExtension(
+          gd,
+          require('./extensions/EBAsyncAction.json'),
+          gdjs,
+          runtimeScene
+        );
+
+      const {
+        free: { wait },
```

Comment:
Ubernit: "free" is hard to grep in the codebase and for a reader without the context it's not obvious what's inside this thing.
```suggestion
        freeFunctions: { wait },
```